app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -76,12 +76,12 @@ $include-html-global-classes: $include-html-classes;
 // b. Grid
 // - - - - - - - - - - - - - - - - - - - - - - - - -
 
-// $include-html-grid-classes: $include-html-classes;
-// $include-xl-html-grid-classes: false;
+$include-html-grid-classes: $include-html-classes;
+$include-xl-html-grid-classes: false;
 
-// $row-width: rem-calc(1000);
-// $total-columns: 12;
-// $column-gutter: rem-calc(30);
+$row-width: rem-calc(1000);
+$total-columns: 12;
+$column-gutter: rem-calc(30);
```

Comment:
These shouldn't be uncommented since you haven't changed anything! You can use the grid system and everything just fine with these commented out!